contribs/gmf/apps/desktop/js/controller.js

```diff
@@ -19,11 +19,6 @@ goog.require('gmf.authenticationDirective');
 goog.require('ngeo.proj.EPSG21781');
 
 
-app.module.constant(
```

Comment:
Why this removal ? Just one uselesss thing that you has seeing (but no relation with this PR) ?